ros2_foxglove_bridge/tests/json_to_ros_test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+
+#include <foxglove_bridge/json_to_ros.hpp>
+
+TEST(JsonToRosTest, EmptyStringMsg) {
+  const std::string payload = "{}";
+
+  auto babelFish = ros2_babel_fish::BabelFish::make_shared();
+
+  ros2_babel_fish::CompoundMessage::SharedPtr output;
+  auto res = foxglove_bridge::jsonMessageToRos(payload, "std_msgs/msg/String", babelFish, output);
```

Comment:
Can you add a test for a more complex message type? E.g. `geometry_msgs/PoseStamped` or something similar, ideally containing an array.